rocky/katalogus/client.py

```diff
@@ -99,8 +103,8 @@ def get_plugin_schema(self, plugin_id) -> Optional[Dict]:
         try:
             Draft202012Validator.check_schema(schema)
             return schema
-        except SchemaError:
-            logger.warning("Invalid schema found for plugin %s", plugin_id)
+        except SchemaError as error:
+            logger.warning("Invalid schema found for plugin %s, %s", plugin_id, error)
```

Comment:
If you intended to only show the error message in the exception (which generally is indeed casting an exception to a string) then this is the right way. If you wanted to show the traceback, it's better to call a logger method (such as `warning` or `info`) with the parameter `exc_info=True` (or `exc_info=error`)

Reply:
At the moment, the schema's all seem to be invalid, and filling up the logs, just letting the user know roughly what went wrong in the schema is enough.